app/models/blacklight_config/default_configuration.rb

```diff
@@ -3,6 +3,10 @@ class DefaultConfiguration
     pattr_initialize :configuration
     def run
       configuration.default_solr_params = default_params
+      configuration.index.title_field = 'title_tesim'
+      configuration.add_index_field Solrizer.solr_name('date', :stored_searchable, type: :string), :label => 'Date'
+      configuration.add_index_field Solrizer.solr_name('description', :stored_searchable, type: :string), :label => 'Description'
+
```

Comment:
Can this be put in its own configuration model? possibly index_configuration.rb?